recipes/clean-up.rb

```diff
@@ -0,0 +1,7 @@
+
+handler_directory = ::File.join(Chef::Config[:file_cache_path], 'handler')
+
+directory handler_directory do
+  recursive true
+  action :delete
+end
```

Comment:
oooh good points! i didn't know about the naming thing, so i'll definitely clean that up if we go forward with this.  i'm not certain we need to be deleting anything, I was just trying to make things cleaner....those are some good issues you brought up... @chris-rock, what's your opinion on this?